src/settings/settingsdialog.cpp

```diff
@@ -98,6 +98,10 @@ SettingsDialog::SettingsDialog(QWidget *parent)
     ui->popupHeightSlider->setMinimum(200);
     ui->popupHeightSpinBox->setMinimum(200);
 
+    // Set maximum and minimum values for the time of the translation notification display
+    ui->translationNotificationDisplayTimeSpinBox->setMaximum(60);
+    ui->translationNotificationDisplayTimeSpinBox->setMinimum(1);
```

Comment:
Maybe set this parameters in `.ui` file?

Reply:
ok.